tests/functional/test_libraries.py

```diff
@@ -8,15 +8,17 @@
 #-----------------------------------------------------------------------------
 
 
+# Library imports
+# ---------------
 import os
 import pytest
+import shutil
 
+# Local imports
+# -------------
 from PyInstaller.compat import is_win
 from PyInstaller.utils.tests import importorskip, xfail_py2
-
-
-# Directory with data for some tests.
-_DATA_DIR = os.path.join(os.path.dirname(os.path.abspath(__file__)), 'data')
+from tests.functional.conftest import DATA_DIR
```

Comment:
The fixture route was surprisingly simple. @codewarrior0, thanks again for the help. Great idea!